docs/mastering-plone/volto_custom_addon2.md

```diff
@@ -551,41 +551,6 @@ Run
 yarn start
 ```
 
-You see
-
-```text
-Module not found: Can't resolve '@eeacms/volto-blocks-form/components'
-```
-
-Why is this? We want the accordion to be reorderable and use the `DragDropList` component of another add-on: `@eeacms/volto-blocks-form`. Add it to the dependencies of your add-on.
-
-{file}`package.json`
-
-```json
-"dependencies": {
-  "@eeacms/volto-blocks-form": "@eeacms/volto-blocks-form"
-},
-```
-
-The following might change the next time:
-
-Add to your **apps** {file}`package.json`:
-
-```json
-"addons": ["@greenthumb/volto-custom-addon", "@eeacms/volto-blocks-form"],
-```
-
-Compile and start your project's app:
-
-```shell
-yarn
-yarn start
-```
-
-```{figure} _static/volto_addon_accordion_add.png
-:alt: "@rohberg/volto-accordion-block"
-```
```

Comment:
I would like to keep the figure.

````
```{figure} _static/volto_addon_accordion_add.png
:alt: "@rohberg/volto-accordion-block"
```
````

Reply:
Oops, the figure was accidentally deleted.